lib/fortnite-api.js

```diff
@@ -417,6 +421,127 @@ class FortniteAPI {
         const uri = endpoints.getLootDetails(id, lang);
         return this.request(uri);
     }
+
+    /**
+     * listSets()
+     * List all the sets used by cosmetics.
+     *
+     * @param {Object} [options]
+     * @param {String} [options.lang] en, ar, de, es, es-419, fr, it, ja, ko, pl, pt-BR, ru, tr
+     * @returns {Promise}
+     */
+
+    listSets(options = {}) {
+        const lang = options.lang || this.defaultLang;
+        const uri = endpoints.listSets(lang);
+        return this.request(uri);
+    }
+
+    /**
+     * getReplayDownloadLink()
+     * Get creative map informations
+     *
+     * @premium
+     * @param {String} id of session
+     * @returns {Promise}
+     */
+    getReplayDownloadLink(id) {
+        const uri = endpoints.getReplayDownloadLink(id);
+        return this.request(uri);
+    }
+
+    /**
+     * getWeaponsDetails()
+     * Get all stats for a specific loot/weapon item
+     * 
+     * @premium
+     * @param {String} id of weapon
+     * @param {Object} [options]
+     * @param {String} [options.lang] en, ar, de, es, es-419, fr, it, ja, ko, pl, pt-BR, ru, tr
+     * @returns {Promise}
+     */
+    getWeaponsDetails(id, options = {}) {
+        const lang = options.lang || this.defaultLang;
+        const uri = endpoints.getWeaponsDetails(id, lang);
+        return this.request(uri);
+    }
+
+    /**
+     * listWeaponsSpawnChance()
+     * List the spawn chances for each type for a given game mode.
+     *
+     * @premium
+     * @param {String} mode
+     * @returns {Promise}
+     */
+    listWeaponsSpawnChance(mode) {
```

Comment:
Likewise, I would say this should be `listWeaponSpawnChances()`!

Reply:
I thinking that correct.

Reply:
I would say `listWeaponSpawnChances` is the most appropriate name for this method, given the description in the API docs.

Reply:
Okay, all is good :)